src/cpp/client.cpp

```diff
@@ -624,8 +616,8 @@ bool Client::tensor_exists(const std::string& name)
 bool Client::dataset_exists(const std::string& name)
 {
     // Same implementation as for tensors; the next line is NOT a type
```

Comment:
I'm a bit puzzled by this comment

Reply:
yes, this old comment should be removed.  Also fixed the typo above the ``dataset_exists()`` function name.